src/gui/virtualstudio.cpp

```diff
@@ -113,9 +113,21 @@ VirtualStudio::VirtualStudio(bool firstRun, QObject* parent)
     m_inMuted       = settings.value(QStringLiteral("InMuted"), false).toBool();
     m_outMuted      = settings.value(QStringLiteral("OutMuted"), false).toBool();
 #ifdef RT_AUDIO
-    m_useRtAudio     = settings.value(QStringLiteral("Backend"), 1).toInt() == 1;
-    m_inputDevice    = settings.value(QStringLiteral("InputDevice"), "").toString();
-    m_outputDevice   = settings.value(QStringLiteral("OutputDevice"), "").toString();
+    m_useRtAudio   = settings.value(QStringLiteral("Backend"), 1).toInt() == 1;
+    m_inputDevice  = settings.value(QStringLiteral("InputDevice"), "").toString();
+    m_outputDevice = settings.value(QStringLiteral("OutputDevice"), "").toString();
+
+    if (m_inputDevice == QStringLiteral("(default)")) {
+        m_inputDevice = "";
+    }
+    if (m_outputDevice == QStringLiteral("(default)")) {
+        m_outputDevice = "";
+    }
+
+    m_baseInputChannel = settings.value(QStringLiteral("BaseInputChannel"), 1).toInt();
+    m_numInputChannels = settings.value(QStringLiteral("NumInputChannels"), 1).toInt();
+    m_inputMixMode = settings.value(QStringLiteral("InputMixMode"), "Mono").toString();
```

Comment:
There's probably more audio-specific context I don't have around this, but I wonder if this is the best setting to pick with respect to backwards compatibility?

I could be wrong, but my understanding is right now all existing JackTrip app users are sending 2 stereo channels (assuming their device supports it). If our goal is to provide a better experience on bandwidth-limited environments by sending only 1 channel, would a default of `m_inputMixMode = "mix-to-mono"` achieve what we want while also preventing existing users from changing their setup?

I'm also looking at the fact that the Setup view could never be seen by existing users because you've unchecked the "Ask again next time" box. IDK, it feels like there may be a small chance that we "break" an existing user's setup by changing the default option from 2 channels/stereo -> 1 channel/mono.

Reply:
I haven't looked at the code, but agree that we need to have 2 defaults:

1. Existing installs/upgrades: migrate to 2 channels / Stereo (channels 1 & 2)
2. Fresh installs: 1 channel/mono (channel 1)

Reply:
Done here: https://github.com/jacktrip/jacktrip/pull/904/commits/d26741e9d08abd2bf3f2bf1d5d20a5606f56f819